specification/paths/ServiceGroups.json

```diff
@@ -0,0 +1,82 @@
+{
+  "get": {
+    "tags": [
+      "ServiceGroups"
+    ],
+    "security": [
+      {
+        "OAuth2": [
+          "shipments.show",
+          "shipments.manage"
+        ]
+      }
+    ],
+    "summary": "Get all available service groups for your API client.",
```

Comment:
The summary is the same as every other `GET` endpoint. I figured it would be better for consistency if we keep the wording the same. And if we decide to change it, it's easier to find all the places where the wording is incorrect.

Reply:
The summary is the same as every other `GET` endpoint. I figured it would be better for consistency if we keep the wording the same. And if we decide to change it, it's easier to find all the places where the wording is incorrect.